app/uk/gov/hmrc/trusts/models/TrustRegistrationModels.scala

```diff
@@ -395,41 +437,54 @@ object NonUKType {
 
 case class PropertyLandType(buildingLandName: Option[String],
                             address: Option[AddressType],
-                            valueFull: Long,
-                            valuePrevious: Long)
+                            valueFull: Option[Long],        // now optional with 5MLD
+                            valuePrevious: Option[Long])    // now optional with 5MLD
 
 object PropertyLandType {
   implicit val propertyLandTypeFormat: Format[PropertyLandType] = Json.format[PropertyLandType]
 }
 
 case class BusinessAssetType(orgName: String,
                              businessDescription: Option[String],
-                             address: AddressType,
-                             businessValue: Long)
+                             address: Option[AddressType],  // now optional with 5MLD
```

Comment:
Same with BusinessAssetType, if they have this asset type then the frontend will always provide an address and value.